app/views/fear_quiz/new.html.erb

```diff
@@ -1,49 +1,334 @@
+          parents
+        <ins><i>parents</i></ins>
+      </label>
+      <input id='thirty-three' type='checkbox' />
+      <label for='thirty-three'>
+        <span></span>
+          rabbits
+        <ins><i>rabbits</i></ins>
+      </label>
+      <input id='thirty-four' type='checkbox' />
+      <label for='thirty-four'>
+        <span></span>
+          rats
+        <ins><i>rats</i></ins>
+      </label>
+      <input id='thirty-five' type='checkbox' />
+      <label for='thirty-five'>
+        <span></span>
+          religion
+        <ins><i>religion</i></ins>
+      </label>
+      <input id='thirty-six' type='checkbox' />
+      <label for='thirty-six'>
+        <span></span>
+          romance
+        <ins><i>romance</i></ins>
+      </label>
+      <input id='thirty-seven' type='checkbox' />
+      <label for='thirty-seven'>
+        <span></span>
+          sharks
+        <ins><i>sharks</i></ins>
+      </label>
+      <input id='thirty-eight' type='checkbox' />
+      <label for='thirty-eight'>
+        <span></span>
+          slender man
+        <ins><i>slenderman</i></ins>
+      </label>
+      <input id='thirty-nine' type='checkbox' />
+      <label for='thirty-nine'>
+        <span></span>
+          snakes
+        <ins><i>snakes</i></ins>
+      </label>
+      <input id='forty' type='checkbox' />
+      <label for='forty'>
+        <span></span>
+          space
+        <ins><i>space</i></ins>
+      </label>
+        </td>
+        <td>
+        <input id='forty-one' type='checkbox' />
+      <label for='forty-one'>
+        <span></span>
+          spiders
+        <ins><i>spiders</i></ins>
+      </label>
+      <input id='forty-two' type='checkbox' />
+      <label for='forty-three'>
+        <span></span>
+          sports
+        <ins><i>sports</i></ins>
+      </label>
+      <input id='forty-four' type='checkbox' />
+      <label for='forty-four'>
+        <span></span>
+          suburban
+        <ins><i>suburban</i></ins>
+      </label>
+      <input id='forty-five' type='checkbox' />
+      <label for='forty-five'>
+        <span></span>
+          teenager
+        <ins><i>teenager</i></ins>
+      </label>
+      <input id='forty-six' type='checkbox' />
+      <label for='forty-six'>
+        <span></span>
+          trash
+        <ins><i>trash</i></ins>
+      </label>
+      <input id='forty-seven' type='checkbox' />
+      <label for='forty-seven'>
+        <span></span>
+          war
+        <ins><i>war</i></ins>
+      </label>
+      <input id='forty-eight' type='checkbox' />
+      <label for='forty-eight'>
+        <span></span>
+          water
+        <ins><i>water</i></ins>
+      </label>
+        <input id='forty-nine' type='checkbox' />
+      <label for='forty-nine'>
+        <span></span>
+          worms
+        <ins><i>worms</i></ins>
+      </label>
+      <input id='fifty' type='checkbox' />
+      <label for='fifty'>
+        <span></span>
+          zombies
```

Comment:
adding a fear will take quite a bit of busy work, unless your fear starts with `z`

Reply:
Good to know that tables work. I was trying to figure out how to wrap thing in containers and such. This is a nice work around.